main.tf

```diff
@@ -1187,6 +1187,8 @@ data "aws_iam_policy_document" "aws_load_balancer_controller" {
       "elasticloadbalancing:DescribeTags",
       "elasticloadbalancing:DescribeTrustStores",
       "elasticloadbalancing:DescribeListenerAttributes",
+      "elasticloadbalancing:DescribeCapacityReservation",
+      "elasticloadbalancing:ModifyCapacityReservation",
```

Comment:
this permission should be added here https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/blob/e94211f6da33687d15fea3aa1186ac9cdcea7794/main.tf#L1353

Reply:
done 👍